full-service/src/db/account.rs

```diff
@@ -964,6 +986,7 @@ mod tests {
                 "Alice's Main Account",
                 "".to_string(),
                 "".to_string(),
+                None,
```

Comment:
Should these fields be commented to indicate which values they are initializing?

Reply:
I'm ok with leaving these uncommented, with the assumption that most of us are using an IDE that will expand the field name in the text view